Islandora-CLAW/Crayfish
# GitHub actions

**GitHub Issue**: (https://github.com/Islandora/documentation/issues/1798)


# What does this Pull Request do?
Converts to github actions
updates the badges
removes travis

# What's new?
* new CI runner

# How should this be tested?
* make sure the tests pass (i don't think they will until this PR is merged - but you can see them running over on the branch itself)
passed here: 
https://github.com/asulibraries/Crayfish/runs/2462160374?check_suite_focus=true#step:9:42
and codecov should be running as well (proof here https://codecov.io/gh/asulibraries/Crayfish/commit/67a862c0d4e086d22f2496e06e0e82e52728b6b7/)

# Interested parties
@Islandora/8-x-committers
